### ISSUE TITLE
TensorFlow 2.x support missing, build broken

### ISSUE THREAD

**Issue by romeokienzler** [Trusted-AI/AIX360#115]
It seems that somebody tried to migrate AIX360 to TensorFlow 2.3.0 but the build is broken - is there somebody who could look into this?

**Comment by vijay-arya** [comment]
@romeokienzler you can take a look at this file, we are on tensorflow 1.14
https://github.com/Trusted-AI/AIX360/blob/master/setup.py
Are you facing any installation issues?

**Comment by romeokienzler** [comment]
yes, I'm on TensorFlow 2.4.0 and want to use LAME with a AIX360 v0.2.1. So now I'm using lame v0.2.0.1 directly, which supports TensorFlow 2.4.0. That's why I was asking if there are any plans to move aix360 to TF 2.x as all my model code is on 2.x
